Getting-Started/Managing-an-Umbraco-project/index.md

```diff
@@ -0,0 +1,31 @@
+---
+meta.Title: "Managing an Umbraco project" 
+meta.Description: "This section allows you to investigate the deployment process, the commercial options and how to best plan out an Umbraco project."
+versionFrom: 8.0.0
+---
+
+# Managing an Umbraco project
+
+This section allows you to investigate the development process, the commercial options and how to best plan out an Umbraco project.
+
+An Umbraco project is the process of developing a website, program or application including the Content Managing System (CMS) which you can use to customise.
+
+In this section you will find information on the key concepts and requirements of working with Umbraco CMS. You will find advise on everything from how to lead a project, how to work with design to the process of launching the completed website.
+
+[What is Umbraco?](https://umbraco/products/umbraco-cms)
+
+[Is Umbraco right for me?](https://umbraco.com/why-choose-umbraco/)
+
+[What commerical options are available from Umbraco?](https://umbraco.com/products/)
+
+[Information on planning an Umbraco project](Setup/Requirements/)
+
+[How do I manage users with Umbraco?](Data/Users/)
+
+[What is the deployment process for Umbraco?](Umbraco-Cloud/Deployment)
+
+[What is the Umbraco Community?](https://community.umbraco.com)
```

Comment:
Would it make sense to somehow group these links under some headings? Just to split up the page a little bit 🤔